BitManipulation/subsets_of_set.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+void possibleSubsets(char A[], int N)
+{
+    for(int i = 0;i < (1 << N); ++i)
+    {
+        for(int j = 0;j < N;++j)
+            if(i & (1 << j))
```

Comment:
Can you explain line 9?

Reply:
This is Pseudo Code. Hope this helps
possibleSubsets(A, N):
    for i = 0 to 2^N:
        for j = 0 to N:
            if jth bit is set in i:
                print A[j]
        print ‘\n’